### PR TITLE
update readme file

### PR DESCRIPTION
**Pull request by areejosama**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,7 @@
-# MirtasApp
+Our app is help Alzeimer patient by using memory card game, plus anyone can play this game to detect if your mental is health or susecptible 
 
-This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 16.1.4.
+Depends on many factors will help in detecing the Alzeimer like the gender, age and game score.
 
-## Development server
+The game has 3 levels, each level will be a bit complicated then the previous one.
```
Comment (yousefalaqra) [inline README.md]: The game roles should be 6 phases

Comment (areejosama) [reply README.md]: Updated, please check

**Comment by yousefalaqra** [comment]
The changes looks great to me, please squash your commits

**Comment by areejosama** [comment]
Hi, I'm trying to squash the commits, but there was a conflict and I fixed it 
so can you please review and accept it so I can continue the squash? Thank you

**Comment by yousefalaqra** [comment]
@areejosama I the squash, should result in combining all commits into one single commit, the history shows 3 commits.

**Comment by areejosama** [comment]
regarding the readme file yes 3 commits, the last one need your review so after that i can merge them, but for now i merged the accepted commits (update readme file, and new update-readme file) in one commit.

**Comment by yousefalaqra** [comment]
@areejosama Please refer to this video as it provides a clear and comprehensive explanation of the squash process. It presents the information in an easy-to-understand manner.

https://www.youtube.com/watch?v=2E23I9PzplM&ab_channel=AndrewConnell

**Review by yousefalaqra — APPROVED** [review]
LGTM! Please go ahead with the merge!

**Pull request merged**